tasks/consul_services_registration.yml

```diff
@@ -7,9 +7,22 @@
     http: "{{ item.http | default (omit) }}" # HTTP endpoint health check. Requires interval.
     script: "{{ item.script | default (omit) }}" # Script/command to check the health of the service. Requires interval.
     interval: "{{ item.interval | default (omit) }}" # Interval between checks.
+    tags: "{{ item.tags | default (omit) }}"
     token: "{{ consul_acl_agent_token }}"
   notify: restart consul
   # We set this because of idempotency issues https://github.com/ansible/ansible-modules-extras/issues/1316
   changed_when: false
   with_items:
     - "{{ consul_services_register }}"
+
+# Temporary, until consul state=absent is fixed
+- name: Consul | De-register services
+  uri:
+    url: "http://localhost:8500/v1/agent/service/deregister/{{ item.name }}"
```

Comment:
At least '8500' should be replaced using 'consul_http_port' instead

Reply:
Done